hsctl

```diff
@@ -26,8 +26,8 @@ HS_DOCKER_IMAGES=(hydroshare_hydroshare hydroshare_defaultworker)
 OTHER_DOCKER_CONTAINERS=(postgis redpanda redis solr)
 
 ### Pre-flight Variables ###
-DEV_SERVER='runuser -p -u hydro-service -g storage-hydro ./run-server'
-DEV_SSH_COMMAND='runuser -p -u hydro-service -g storage-hydro /usr/sbin/sshd'
+DEV_SERVER='runuser -p ./run-server'
```

Comment:
@pkdash I requested your review on this PR mostly because I think you're the only one who still uses `hsctl` and I want to make sure that the PR doesn't break it for you.

Thanks!